src/prompts/alert/alert.scss

```diff
@@ -0,0 +1,114 @@
+@import '../common.scss';
+
+.kumulos-alert-container {
+    transition: transform 0.2s $quintEasing, box-shadow 0.2s $quintEasing;
+    will-change: transform, box-shadow, width, height;
+
+    background: #fff;
+
+    position: fixed;
+
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    max-height: 100%;
+    padding: 20px;
+
+    display: flex;
+    flex-direction: column;
+
+    font-family: helvetica, sans-serif;
```

Comment:
Can we line up and use same font stack declared on Overlay? Could even perhaps extract to var or something.

Reply:
Already same fonts, will abstract to variable

Reply:
Done